### PR TITLE
Creating documentation for Lightweight access token

### LINKED ISSUE #23725: Documentation for lightweight access token
### Description

We may want to add something to the documentation about lightweight access token. We can probably do it after finish Milestone 2 from the epic #21186 . Documentation then might be slightly updated after finish also milestone 3.

### PR DESCRIPTION
**Pull request by skabano**
Lightweight access token support follows milestone #21186.
Since M0,1 have already been implemented, I will create documentation of milestone 2.
Details of the M2 documentation can be found in #23725.

closes #23725

### CODE REVIEW

**Comment by skabano** [comment]
@mposolda 
The lightweight access token was described in the Server Administration Guide.
Screenshots are under construction.

File: docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc [hunk 1]
```diff
@@ -66,3 +66,16 @@ Service account sessions provide the following details:
...
+Further, when the resource server acquires the PII removed from the access token, it can acquire the PII by sending the access token to {project_name}'s token introspection endpoint.
+
+Information that cannot be removed from a lightweight access token::
+  Protocol mappers can controls which information is put onto an access token and the lightweight access token use the protocol mappers. Therefore, the following information cannot be removed from the lightweight access. +
+  `exp`, `iat`, `auth_time`, `jti`, `iss`, `sub`, `typ`, `azp`, `nonce`, `session_state`, `sid`, `scope`, `cnf`
+
+Using a lightweight access token in {project_name}::
+  By applying `use-lightweight-access-token` executor of client policies to a client, the client can receive a lightweight access token instead of an access token. The lightweight access token contains a claim controlled by a protocol mapper where its setting `Add to lightweight access token`(default OFF) is turned ON. Also, by turning ON its setting `Add to token introspection` of the protocol mapper, the client can obtain the claim by sending the access token to {project_name}'s token introspection endpoint.
```
Comment (mposolda) [inline docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc:81]: Is it possible to add link to the "Client Policies" section here (or "Client policies executor section"), so it is easier for administrator to know how to configure client policies ?

**Review by mposolda — COMMENTED** [review]
@skabano Thanks! Added one minor comment inline. Also adding @andymunro to review the documentation.
@andymunro Could you please review this PR?

File: docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc [hunk 2]
```diff
@@ -66,3 +66,16 @@ Service account sessions provide the following details:
 Use the *Script Mapper* to map claims to tokens by running user-defined JavaScript code. For more details about deploying scripts to the server, see link:{developerguide_jsproviders_link}[{developerguide_jsproviders_name}].
 
 When scripts deploy, you should be able to select the deployed scripts from the list of available mappers.
+
+[[_using_lightweight_access_token]]
+== Using lightweight access token
+The access token in {project_name} contains sensitive information, including Personal Identifiable Information (PII).
+Therefore, to satisfy the use case that the resource server does not want to disclose such the information to third party entities such as clients, {project_name} supports lightweight access tokens that remove PII from access tokens.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc:73]:
```suggestion
Therefore, if the resource server does not want to disclose this type of information to third party entities such as clients, {project_name} supports lightweight access tokens that remove PII from access tokens.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
A few changes to suggest, @skabano

File: docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc (hunk 1 above)
Comment (skabano) [reply docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc]: @mposolda 
Thank you for your comment.
Added link to the "Client Policies" section.

**Comment by skabano** [comment]
> A few changes to suggest, @skabano

@andymunro 
Thank you for your comment.
As suggested, I have corrected it.

**Review by mposolda — APPROVED** [review]
@skabano Thanks!
@andymunro Thanks for the review!

**Pull request merged**